src/GitVersion.Core/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,48 @@
+abstract GitVersion.VersionCalculation.VersionStrategyBase.GetBaseVersions(GitVersion.Model.Configuration.EffectiveBranchConfiguration! configuration) -> System.Collections.Generic.IEnumerable<GitVersion.VersionCalculation.BaseVersion!>!
```

Comment:
Most of these should probably be moved to `.Shipped`, no?

Reply:
Actually I'm not certain what the pattern is behind it. So I just try to fix the warnings. ;) If you like I can move it to Shipped never mind.

Reply:
If I'm not mistaken, these files are artifacts from `Microsoft.CodeAnalysis.PublicApiAnalyzers`. [You can read more about it here](https://www.drunkonmonads.com/tracking-breaking-changes/).

Reply:
Done.